.ci/rerun-workflow.sh

```diff
@@ -31,17 +31,17 @@ pr_branch=$3
 runs=$(
   curl -sL \
     -H "Accept: application/vnd.github.v3+json" \
-    -H "Authorization: token $SECRETS_WORKFLOW" \
+    -H "Authorization: token ${SECRETS_WORKFLOW}" \
```

Comment:
Was this intended to address a specific `shellcheck` error, or is it just something you're proposing for stylistic similarity? I don't mind this change (I generally support wrapping interpolation in `${}` like this), just trying to understand the change.

Reply:
ok just checking. This is fine, let's leave it (I do prefer the style with `${}`) but please don't touch other lines that aren't directly related to the purpose of the PR.